insights/client/apps/compliance/__init__.py

```diff
@@ -22,9 +23,10 @@ def __init__(self, config):
         self.hostname = get_canonical_facts().get('fqdn', '')
 
     def oscap_scan(self):
+        self._assert_oscap_rpms_exist()
         policies = self.get_policies()
         if not policies:
-            logger.error("ERROR: System does not exist!\n")
+            logger.error("System is not associated with any profiles. Assign profiles by either uploading a SCAP scan or using the compliance web UI.\n")
```

Comment:
@katierik UX check on this error message. This occurs when the system is registered with insights, but there are no associated profiles in the compliance app.

Reply:
This is a huge improvement. What did the user do to get here?

Reply:
This is one of the failure modes of running `insights-client --compliance`. In order for the client to run a SCAP scan against a policy, the client needs to query the compliance app for associated policies for this host. If there are no policies associated with this host, this is the error message displayed. Feel free to reach out to me if you still have questions.